src/megachatapi_impl.cpp

```diff
@@ -1426,6 +1435,13 @@ void MegaChatApiImpl::sendPendingRequests()
                 break;
             }
 
+            if (chatroom->publicChat() && chatroom->numMembers() > MAX_PUBLICCHAT_MEMBERS_FOR_CALL)
+            {
+                API_LOG_ERROR("Start call - the public chat has too many participants");
+                errorCode = MegaChatError::ERROR_ACCESS;
```

Comment:
Why not `ETOOMANY` for calls, similar than to make-private? (here and also at `ANSWER_CALL`

Reply:
Fixed